instrumentation/akka-http-10.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkahttp/client/AkkaHttpClientSingletons.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.akkahttp.client;
+
+import akka.http.scaladsl.model.HttpRequest;
+import akka.http.scaladsl.model.HttpResponse;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanStatusExtractor;
+import io.opentelemetry.javaagent.instrumentation.akkahttp.AkkaHttpUtil;
+
+public class AkkaHttpClientSingletons {
+
+  private static final HttpHeaderSetter SETTER;
+  private static final Instrumenter<HttpRequest, HttpResponse> INSTRUMENTER;
+
+  static {
+    SETTER = new HttpHeaderSetter(GlobalOpenTelemetry.getPropagators());
+    AkkaHttpClientAttributesExtractor httpAttributesExtractor =
+        new AkkaHttpClientAttributesExtractor();
+    INSTRUMENTER =
+        Instrumenter.<HttpRequest, HttpResponse>newBuilder(
+                GlobalOpenTelemetry.get(),
+                AkkaHttpUtil.instrumentationName(),
+                HttpSpanNameExtractor.create(httpAttributesExtractor))
+            .setSpanStatusExtractor(HttpSpanStatusExtractor.create(httpAttributesExtractor))
+            .addAttributesExtractor(httpAttributesExtractor)
+            .addAttributesExtractor(new AkkaHttpNetAttributesExtractor())
```

Comment:
```suggestion
            .addAttributesExtractor(new AkkaHttpNetAttributesExtractor())
            .addRequestMetrics(HttpClientMetrics.get())
```

Reply:
Also, can you add `PeerServiceAttributesExtractor` to the client instrumenter too?